src/register-org/components/company-house-details/company-house-details.component.html

```diff
@@ -0,0 +1,43 @@
+<main id="content" class="govuk-main-wrapper" role="main" tabindex="-1">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <a class="govuk-back-link" href="javascript:history.back()">Back</a>
+      <span class="govuk-caption-xl">Register an organisation</span>
+      <h1 class="govuk-heading-xl">What is your company name and Companies House number?</h1>
+      <hr>
+      <p class="govuk-body">
+        If your company is registered with Companies House,
+        add the company name and number as they appear in the
+        <a class="govuk-link" href="https://www.gov.uk/get-information-about-a-company" target="_blank">company
+          information register</a>
+      </p>
+      <form>
```

Comment:
Shouldn't this be linked to a formGroup if you're using formControlNames on the inputs?

Reply:
@ronaldmansveld - this was a display-only ticket integration for this page. The functional ticket will be coming later in the sprint.